python/src/ExpectationSimulationAlgorithm_doc.i.in

```diff
@@ -11,70 +11,268 @@ X : :class:`~openturns.RandomVector`
-  - The norm-1: :math:`\frac{1}{d} \sum_1^{d} |\sigma_i| \leq \max_{\sigma}`
-  - The norm-2: :math:`\sqrt{\frac{1}{d} \sum_1^{d} \sigma_i^2} \leq \max_{\sigma}`
+- the maximum (the default): 
+
+.. math::
+
+    \max_{i=1,\ldots,n_X} \hat{\sigma}_i / \sqrt{n} \leq \max_{\sigma},
+
+- the norm-1: 
+
+.. math::
+
+    \sum_{i=1}^{n_X} \left|\hat{\sigma}_i / \sqrt{n}\right| \leq \max_{\sigma},
+
+- the norm-2: 
+
+.. math::
+
+    \sqrt{\sum_{i=1}^{n_X} \left(\hat{\sigma}_i / \sqrt{n} \right)^2} \leq \max_{\sigma},
+
+- or disabled. 
+
+The type of operator on the coefficient of variation can be set using :meth:`setStandardDeviationCriterionType`.
+
+The default type is set by the `ExpectationSimulationAlgorithm-DefaultStandardDeviationCriterionType` key of the :class:`~openturns.ResourceMap`.
+
+The threshold :math:`\max_{\sigma}` can be set using :meth:`setMaximumStandardDeviation`.
 
-  The type of operator on the coefficient of variation can be set using :meth:`setStandardDeviationCriterionType`.
+Let :math:`\max_{\sigma_1}, \ldots, \max_{\sigma_{n_X}}` be the 
+componentwise maximum standard deviations. 
+The criterion on the maximum deviation per component is defined by:
 
-  The default type is set by the `ExpectationSimulationAlgorithm-DefaultStandardDeviationCriterionType`
-  :class:`~openturns.ResourceMap` key.
+.. math::
 
-  The threshold :math:`\max_{\sigma}` can be set using :meth:`setMaximumStandardDeviation`.
+    \sigma_i \leq \max_{\sigma_i}
 
-The criterion on the maximum deviation per component is defined by :math:`\sigma_i \leq \max_{\sigma_i}`
+for :math:`i = 1, \ldots, n_X`.
 
-  The thresholds :math:`\max_{\sigma_i}` can be set using :meth:`setMaximumStandardDeviationPerComponent`.
+The threshold vector :math:`\max_{\sigma_i}` can be set using :meth:`setMaximumStandardDeviationPerComponent`.
 
-  By default this criterion is disabled.
+By default this criterion is disabled.
 
+The default values of the parameters are based on the 
+following keys of the :class:`~openturns.ResourceMap`:
+
+- `SimulationAlgorithm-DefaultMaximumOuterSampling`,
+- `SimulationAlgorithm-DefaultMaximumCoefficientOfVariation`,
+- `SimulationAlgorithm-DefaultMaximumStandardDeviation`.
+
+In general, the criteria that should be used is based on the coefficient of variation (C.O.V.) because it guarantees a relative accuracy on the estimate of the mean. However, it may happen that we estimate an expected value which is equal to zero (perhaps for teaching or research purposes). The default value of `SimulationAlgorithm-DefaultMaximumStandardDeviation` is so that the standard deviation criteria is disabled by default (in order to let the C.O.V. be used by default). If the expected value of :math:`\vect{X}` is zero, the algorithm always reach the maximum number of outer iterations. If this happens, please configure the maximum standard deviation in order to match the scale of the random variable at hand.
```

Comment:
```suggestion
In general, criteria based on coefficients of variation (C.O.V.) should be used because they guarantee a relative accuracy on the estimate of the mean. However, we may happen to estimate an expected value equal to zero (perhaps for teaching or research purposes). The default value of `SimulationAlgorithm-DefaultMaximumStandardDeviation` is `0`, which means that standard deviation-based criteria are disabled by default (in order to let C.O.V.-based criteria be used by default). If the expected value of :math:`\vect{X}` is zero, the algorithm is likely to reach the maximum number of outer iterations. If this happens, please configure the maximum standard deviation in order to match the scale of the random variable at hand.
```
Several style suggestions, but the main point is that the algorithm will not "always" reach the maximum of outer iterations if the expected value is 0. You may get a weird, unlikely sample from time to time.